<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>errorLogWidget</class>
 <widget class="QWidget" name="errorLogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>506</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Error Log</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="11">
    <widget class="QTableView" name="logTable">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>140</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="errorLogShow">
     <property name="text">
      <string>&amp;Show</string>
     </property>
     <property name="buddy">
      <cstring>errorLogComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="errorLogComboBox">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ERROR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WARNING</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UI-WARNING</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FONT-WARNING</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EXPORT-WARNING</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EXPORT-ERROR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TRACE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DEPRECATED</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
  <action name="actionRowSelected">
   <property name="text">
    <string>RowSelected</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>errorLogComboBox</tabstop>
  <tabstop>logTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
